spynnaker/pyNN/models/populations/population.py

```diff
@@ -569,8 +569,10 @@ def initial_values(self):
                 "Population does not support the initialisation")
         return self._vertex.initial_values
 
-    # NON-PYNN API CALL
     def get_initial_value(self, variable, selector=None):
+        raise NotImplementedError("use initial_values property")
```

Comment:
shouldnt we add a depreciated thingy here as well, so next release we can kill it fully

Reply:
Yes could do.
What format do you suggest the takes.